internal/webservice/routes.go

```diff
@@ -348,10 +348,25 @@ func (ws *WebService) getAppsPerPartitionPerQueue(req *restful.Request, resp *re
 	jsonResponse(resp, daoApps)
 }
 
+// getNodesPerPartition returns all nodes for a given partition and queue.
+// Results are ordered by creation time in descending order.
+// Following query params are supported:
+// - nodeId: filter by nodeId
```

Comment:
As you do with `schedulable` and `isReserved` down below, maybe note the parameter type (e.g. string, int, etc) on these other parameters, to be consistent?

Reply:
Sure, updated. btw these comments are temporarily added aded for the front until we come up with some sort of documentation.